src/Step/Deploy/InstallUpdate/ConfigUpdate/DbConnection.php

```diff
@@ -163,10 +166,10 @@ public function execute()
         );
 
         if (!empty($customSplitConnections)) {
-            $this->logger->warning(sprintf(
-                'For split databases used custom connections: %s',
-                implode(', ', $customSplitConnections)
-            ));
+            $this->logger->warning(
+                sprintf('For split databases used custom connections: %s', implode(', ', $customSplitConnections)),
```

Comment:
Not sure what this message means --  split databases configured with custom connections? -- or the split database connections have already been configured (used)?